package.json

```diff
@@ -310,7 +311,7 @@
     "query-string": "4.1.0",
     "raven": "2.6.4",
     "raven-js": "3.27.2",
-    "razzle": "3.1.2",
+    "razzle": "^3.1.5",
```

Comment:
@tiberiuichim can we pin this specifically to 3.1.5? I prefer to have exactly pinned versions to have a repeatable CI run.